From 8f22923341d4c89c63c5aeac4537439dbb31a821 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Mon, 1 Jul 2019 17:59:48 +0700
Subject: [PATCH 026/268] arm64: dts: renesas: r8a774c0-cat874: enable USB3.0
 function

This patch enables USB3.0 function device for the RZ/G2E board on the
R8A774C0 SoC.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
index 0a52b42..822c991 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
@@ -462,3 +462,9 @@
 
 	status = "okay";
 };
+
+&usb3_peri0 {
+	companion = <&xhci0>;
+
+	status = "okay";
+};
-- 
2.7.4

